source/patterns/@aws-solutions-constructs/aws-lambda-sqs/lib/index.ts

```diff
@@ -114,10 +134,6 @@ export class LambdaToSqs extends Construct {
       defaults.CheckProps(props);
 
       if (props.deployVpc || props.existingVpc) {
-        if (props.deployVpc && props.existingVpc) {
-          throw new Error("More than 1 VPC specified in the properties");
-        }
```

Comment:
Deleted because this check is redundant and can never be hit. This check is already being done from the [defaults.CheckProps call here](https://github.com/awslabs/aws-solutions-constructs/blob/main/source/patterns/%40aws-solutions-constructs/core/lib/input-validation.ts#L168)****